pipeline/extract.py

```diff
@@ -1,12 +1,19 @@
 """This file is responsible for downloading the latest earthquake data"""
+# pylint: disable=W0718
+
 import datetime
+import logging
 
 import requests
 
 URL = "https://earthquake.usgs.gov/earthquakes/feed/v1.0/summary/all_hour.geojson"
 FEATURES = "features"
 PROPERTIES = "properties"
 TIME = "time"
+UPDATED = "updated"
+
+logging.basicConfig(level=logging.INFO,
+                    format="%(asctime)s %(levelname)s %(message)s")
```

Comment:
This could be added into `extract_process`

Reply:
That's true, it looks cleaner that way - I've made this change